test/unit/tasks/download-assets.test.js

```diff
@@ -45,7 +45,7 @@ nock(`https://${API_HOST}`)
   .post(`/spaces/${SPACE_ID}/environments/${ENVIRONMENT_ID}/asset_keys`, {
     expiresAt: /.+/i
   })
-  .times(2)
+  .times(1)
```

Comment:
What is this doing exactly and why?

Reply:
so, it's a mock http service where we expect to receive a call to create a new cache key, however since now we fixed the condition it creates key once and later it uses the cached item, before it was creating a new cache item every time